lib/src/dart_formatter.dart

```diff
@@ -88,13 +88,13 @@ class DartFormatter {
     // TODO(paulberry): consider plumbing in experiment enable flags from the
     // command line.
     var featureSet = FeatureSet.fromEnableFlags2(
-        sdkLanguageVersion: Version(2, 13, 0),
-        flags: [
-          'constructor-tearoffs',
-          'generic-metadata',
-          'nonfunction-type-aliases',
-          'triple-shift'
-        ]);
+      sdkLanguageVersion: Version(2, 16, 0),
```

Comment:
Will this affect the ability for this package to run on older SDKs?

Reply:
Run - definitely no. Support code for older language versions - I don't think it will. These language features are additive.